stacks/root.yml

```diff
@@ -123,6 +123,8 @@ Mappings:
       DovetailMemory: 400
```

Comment:
Nitpicky, but we're trying to follow the [seven rules](https://chris.beams.io/posts/git-commit/#seven-rules) for this repo (capitalize commit messages, etc)

Reply:
Is the issue the commit message, or something else? (I've just read them they are not consistent)